db/schema.rb

```diff
@@ -167,11 +167,8 @@
     t.text "intimidation_to_witnesses_details"
     t.string "hostage_taker"
     t.boolean "staff_hostage_taker"
-    t.date "date_most_recent_staff_hostage_taker_incident"
```

Comment:
This needs to be removed from this PR as it is not relevant and will cause conflicts later on.